1.7.0/values.yaml

```diff
@@ -0,0 +1,157 @@
+# Default values for DirectorOnPrem.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# Update platfrom according to your Platform, Default include(GKE,Packet) and SUSE
+platform: Default
+
+server:
+  release: 1.7.0-ee
+  dockerSecret: directoronprem-registry-secret
+  protocol: http
+  url:
+  httpNodePort: 30380
```

Comment:
should we need this as will be running as DS
cc @atulabhi

Reply:
@shovanmaity it will be required for deployment